### PR TITLE
mvn -ntp

### PR DESCRIPTION
**Pull request by Bananeweizen**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -104,14 +104,16 @@ spec:
 							dir ('eclipse.platform.swt.binaries') {
 								sh '''
 									/opt/tools/apache-maven/latest/bin/mvn install \
-										--batch-mode -Pbuild-individual-bundles -DforceContextQualifier=zzz -Dnative=gtk.linux.x86_64 \
+										--batch-mode --no-transfer-progress
```
Comment (mickaelistria) [inline Jenkinsfile]: Missing `\` at end of line.

File: Jenkinsfile [hunk 2]
```diff
@@ -104,14 +104,16 @@ spec:
...
+										-Pbuild-individual-bundles -DforceContextQualifier=zzz -Dnative=gtk.linux.x86_64 \
 										-Dcompare-version-with-baselines.skip=true -Dmaven.compiler.failOnWarning=true
 								'''
 							}
 							dir ('eclipse.platform.swt') {
 								sh '''
 									/opt/tools/apache-maven/latest/bin/mvn clean verify \
-										--batch-mode -Pbuild-individual-bundles -DcheckAllWS=true -DforkCount=0 \
+										--batch-mode --no-transfer-progress
```
Comment (mickaelistria) [inline Jenkinsfile]: Same.

**Review by mickaelistria — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (Bananeweizen) [reply Jenkinsfile]: 🤦🏻‍♂️ Thanks @mickaelistria . That happens when you try to change something online via the browser..

**Comment by Bananeweizen** [comment]
Please be aware that this will only work if all related build machines (github? Eclipse foundation?) have a recent Maven version.

**Review by mickaelistria — APPROVED** [review]

**Comment by mickaelistria** [comment]
It's OK to merge it right now despite freeze as it's releng configuration (no payload in actual build result).

**Pull request merged**